### ISSUE TITLE
C++ TensorRT Infer by onnx model

### ISSUE THREAD

**Issue by HLX-GitHub** [iMoonLab/Hyper-YOLO#10]
Thank you for the author! In class MessageAgg will generate IsInf node while exporting onnx model which is not supported by TensorRT! I have modified the code into the following form in "block.py" which can be run in TensorRT!

sum_tensor = torch.sum(path, dim=2, keepdim=True)
norm_out = torch.where(sum_tensor != 0, 1 / sum_tensor, sum_tensor)
